mintapi/signIn.py

```diff
@@ -368,10 +369,18 @@ def sign_in(
         )
         account_selection_page(driver, intuit_account)
         password_page(driver, password)
-        # Give the overview page a chance to actually load
-        WebDriverWait(driver, 5).until(
-            expected_conditions.url_contains("https://mint.intuit.com/overview")
-        )
+        # Give the overview page a chance to actually load.
+        # If it doesn't, then there may be another round of MFA.
+        try:
+            WebDriverWait(driver, 5).until(
+                expected_conditions.url_contains("https://mint.intuit.com/overview")
+            )
+        except Exception:
+            count += 1
+            if count > 10:
```

Comment:
Nice! Do we actually want to hold this up for 10 loops (almost a minute)? This was implemented to handle a 2nd loop IIRC, so maybe 3 would be sufficient? Or 4-5?

Reply:
Yeah fair point! I think 4 is good for now.

Reply:
@mrooney done!